apollo/interfaces/agent_response.py

```diff
@@ -56,5 +56,10 @@ def calculate_result_size(self) -> int:
             return 0
         return len(self.serialize_result().encode())
 
-    def serialize_result(self) -> str:
-        return json.dumps(self.result, cls=AgentSerializer)
+    def serialize_result(self, unwrap_result: bool = False) -> str:
```

Comment:
I need the data to be stored without any agent message wrapping, as the data is intended to be used by clients directly.